### ISSUE TITLE
a11y: "Move cursor to beginning/end" doesn't work

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#13469]
On Android these actions are pre-populated in the local context menu (swipe up and left), but currently don't work.

**Comment by goderbauer** [comment]
We probably need to implement ACTION_SET_SELECTION.
